pkg/kubectl/cmd/certificates.go

```diff
@@ -98,28 +119,31 @@ func NewCmdCertificateApprove(f cmdutil.Factory, out io.Writer) *cobra.Command {
 }
 
 func (options *CertificateOptions) RunCertificateApprove(f cmdutil.Factory, out io.Writer, force bool) error {
-	return options.modifyCertificateCondition(f, out, force, func(csr *certificates.CertificateSigningRequest) (*certificates.CertificateSigningRequest, bool, string) {
+	return options.modifyCertificateCondition(f, force, func(csr *certificates.CertificateSigningRequest) (*certificates.CertificateSigningRequest, bool) {
```

Comment:
Fix `options.modifyCertificateCondition` to accept `ClientSet` and `Builder` rather than entire factory.